File: HA5_5.java
//Write a program which accepts N and print first 5 Multiples of it
//Input : 4
//Output : 4    8    12     16      20

import java.util.Scanner;

class Number
{
    public void Pattern(int iNo)
    {
        int iCnt = 0;
        for(iCnt = 1; iCnt <=5; iCnt++)
        {
            System.out.println(iCnt*iNo)
        }
    }
}

public class HA5_5
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter a number ");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        {
            nobj.Pattern(iValue);
        }
        sobj.close();
    }
}

File: HA5_5.ts
//Write a program which accepts N and print first 5 Multiples of it
//Input : 4
//Output : 4    8    12     16      20

import * as readline from 'readline'

function Pattern(iNo : number)
{
    let iCnt : number =0
    for(iCnt = 1; iCnt<=5; iCnt++)
    {
        console.log(iCnt*iNo)
    }
}

async function main()
{
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    })

    let iValue : number = 0

    iValue = parseInt(await askQuestion(rl, "Enter a number")) ||0

    Pattern (iValue)
}
function askQuestion(rl: readline.Interface, question: string): Promise<string>
{
    return new Promise((resolve) =>{
        rl.question(question,(answer : any) => {
            resolve(answer)
        })
    })
}
main()